data/data/nutanix/master/outputs.tf

```
@@ -0,0 +1,3 @@
+output "control_plane_ips" {
+  value = nutanix_virtual_machine.vm_master.*.nic_list_status.0.ip_endpoint_list.0.ip
```

Let's use the new style for splat expressions.
```suggestion
  value = nutanix_virtual_machine.vm_master[*].nic_list_status[0].ip_endpoint_list[0].ip
```